src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al

```diff
@@ -26,8 +26,50 @@ page 1933 "Perf. Profiler Schedules List"
     {
         area(Content)
         {
-            repeater(Profiles)
+            group("Profiling Status")
             {
+                Caption = 'Profiling Status';
+                AboutTitle = 'Profiling Status';
+                AboutText = 'See if profiling is enabled for the current user session.';
+
+                field("Profiling Enabled"; IsProfilingEnabled)
+                {
+                    Caption = 'Profiling Enabled';
+                    ToolTip = 'Shows if profiling is enabled for the current user session.';
```

Comment:
Tooltips should begin with "Specifies".
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/user-assistance#guidelines-for-tooltip-text
(check all other new tooltips as well)

Reply:
Fixed in all the tooltips I could find